index.js

```diff
@@ -0,0 +1,15 @@
+
+function changeColor(hola){
+    hola.style.color = 'red';
+}
+
+
+function hideMenu(){
+    let displayMenu = document.getElementById('id__links')
+    if(displayMenu.style.display === 'none'){
```

Comment:
2 cosas, por convencion el else va pegado al }else, pero te recomiendo evitar usar else. Queda mas legible usar "corto circuito" que es retornar. Entonces si entra al if retorna y no va a la parte de abajo
```suggestion
    if(displayMenu.style.display === 'none'){
           displayMenu.style.display = 'flex'
           return
    }
    displayMenu.style.display = 'none';
```